### ISSUE TITLE
Add alt text to valuation examples

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1661]
Current alt text can be pulled from the live site and the pdf version of the examples.

2016 rule example
Dev: https://dev-onrr-frontend.app.cloud.gov/references/valuation/processed-gas-example
Live site: https://www.onrr.gov/valuation/2016-rule-processed-example.htm

POP example
Dev: https://dev-onrr-frontend.app.cloud.gov/references/valuation/pop-example
Live site: https://www.onrr.gov/valuation/federal-pop-example.htm

**Comment by cthomasONRR** [comment]
@mcharg I can do this. I added issue #1660 because I'm having some formatting issues after adding alt text.

**Comment by cthomasONRR** [comment]
@jlouis-onrr 
I'm trying to add alt text to the 2016 Reporting example (https://dev-onrr-cms.app.cloud.gov/admin/content/pages/68). And it will not save the text added to the 'alt text' box. Every time I click the green check, and go back in, it's gone: 
![image](https://user-images.githubusercontent.com/72526208/155178617-d2fb34ad-d473-4b09-9cf1-dc3fb2d24895.png)

**Comment by cthomasONRR** [comment]
In the POP example the following content blocks will not save the Alt Text:
- [ ] Overview Intro 2 - pop example

**Comment by cthomasONRR** [comment]
@mcharg I was able to finish adding alt text. Could you please review? Every image should have alt text associated to it now.

**Comment by mcharg** [comment]
@cthomasONRR This image is missing alt text.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/21a8b5e7-d0f4-4d7e-9d2d-708fbf9eefa4)

**Comment by mcharg** [comment]
Another one. ![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/692eaf61-1d41-4204-ac0f-1f363848451c)

**Comment by cthomasONRR** [comment]
@mcharg OK I got the plant statement and PC03-9... you posted the PC03-9 image twice. I'm looking for another image that doesn't have alt text but can't find any.

**Comment by mcharg** [comment]
Here's on on the POP example. ![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/808fd551-e0a3-4b45-baf1-baaa68a3c1f8)

**Comment by mcharg** [comment]
Another one on POP.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/958fff91-0c49-42a4-9c44-3dfcccc09d22)

**Comment by mcharg** [comment]
The reason the two on POP are the same is the royalty reporting block is being used on two different expansion panels. ![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/000ee4f9-1764-4941-9c6f-a02ab64b0781)

**Comment by cthomasONRR** [comment]
@mcharg Something is off in your visuals... I don't see that at all: 
![image](https://user-images.githubusercontent.com/72526208/155560212-4419206e-8ca2-4634-b376-770535c5556c.png)

Maybe close out and reopen. This was happening to me when I was building these pages.

**Comment by cthomasONRR** [comment]
![image](https://user-images.githubusercontent.com/72526208/155560411-6afb361b-53ee-4f19-8740-ca21666626b2.png)

**Comment by mcharg** [comment]
Closing out and refreshing fixed the POP ones.  I think this is good to close.
